### DISCUSSION TITLE
Mapping Custom Data Types from Flutterflow to Rowy

### DISCUSSION THREAD

**Discussion by DayneLinx** [AntlerVC/firetable#1493] [Support / Q&A] [answered]
Hi,

I have a few custom Data Types in flutterflow and wanted to know how to map them so they are represented in Rowy. Would I use a Sub-Table for this? 

Custom Data Types

[https://docs.flutterflow.io/data-and-backend/custom-data-types]()

Thanks in advance.

**Comment by Chadha93** [comment, marked as answer]
Hi @DayneLinx, Yes, you can use the Sub-Table for this - https://docs.rowy.io/field-types/sub-table#creating-a-sub-table.
Also to map you can use Array of Maps, here is the perfect documentation with an example of FlutterFlow - https://docs.rowy.io/integrations/flutterflow/map-array.

**Reply by DayneLinx** [reply]
> Hi @DayneLinx, Yes, you can use the Sub-Table for this - https://docs.rowy.io/field-types/sub-table#creating-a-sub-table.
> 
> Also to map you can use Array of Maps, here is the perfect documentation with an example of FlutterFlow - https://docs.rowy.io/integrations/flutterflow/map-array.

Thank you!
